test-target/local-pod-under-test.yaml

```diff
@@ -23,12 +23,11 @@ spec:
     spec:
       automountServiceAccountToken: false
       containers:
-        - command: [ "/bin/bash", "-c", "echo 'logs' && tail -f /dev/null" ]
-          image: quay.io/testnetworkfunction/cnf-test-partner:latest
+        - image: quay.io/testnetworkfunction/cnf-test-partner:latest 
           imagePullPolicy: IfNotPresent
           name: test
           ports:
-            - name: "testport"
+            - name: "probeport"
```

Comment:
Just FYI, verizon wants the port name to be "protocol-name" and we may add that test soon

Reply:
do you you which section this recommendation is mentioned ? is it on 1.3 document ?

Reply:
Yes, in 1.3, section 19.3

Reply:
fixed.